### DISCUSSION TITLE
dpkg install fails on invalid Version field

### DISCUSSION THREAD

**Discussion by lainsw** [glikely/obs-ptz#130] [General]
Hey- I have an install question. If I try to run

`sudo dpkg -i obs-ptz-v0.13.0-linux-x86_64.deb`

I get an error caused by the version number not starting with a digit:

`dpkg: error al procesar el archivo obs-ptz-v0.13.0-linux-x86_64.deb (--install): parsing file '/var/lib/dpkg/tmp.ci/control' near line 8 package 'obs-ptz': 'Version' field value 'v0.13.0': el número de versión no empieza por un dígito Se encontraron errores al procesar: obs-ptz-v0.13.0-linux-x86_64.deb`

(I run the system in Spanish)

Is everyone just copying the files in the .deb somewhere? The manifest seems to scatter them all over /usr/share.../lib/... and so forth.

(fresh install of Ubuntu Studio LTS 22.04 with updated OBS 29.0.0)

**Comment by glikely** [comment]
This should be solved with v0.13.1

there is still a dependency issue with the package though. Make sure you manually install the qt6 serialport package from the Ubuntu repo
